.github/workflows/daily_ci.yml

```diff
@@ -3,7 +3,6 @@ on:
   schedule:
     - cron: '0 8 * * *'
   repository_dispatch:
```

Comment:
Do you think `repository-dispatch:` is still required for the trigger and wait action, since it uses the `.yml` file path as a target instead of a trigger name?

Reply:
I think you are right